hack/allocate.sh

```diff
@@ -305,7 +305,7 @@ namespace() {
   # Default Broker
   $KUBECTL apply -f - <<EOF
   apiVersion: eventing.knative.dev/v1
-  kind: broker
+  kind: Broker
```

Comment:
This should have never been `broker` :sweat_smile: 

Is this now failing, b/c a newer Kube version is less lenient, and hence failing?

Reply:
It most likely fails because of new eventing release.